### PR TITLE
Subscript

### PR DESCRIPTION
**Pull request by CheeseMuffin**
Added subscript for messages, in the form of \\text\\

### CODE REVIEW

**Comment by panpawn** [comment]
No option to suppress it?

**Comment by CheeseMuffin** [comment]
I don't think I can change he showdown_prefs for the access parameters?

**Comment by Zarel** [comment]
I mean, you're using the `options.hidesuperscript` but you could add your own permission to the options screen instead.

**Comment by CheeseMuffin** [comment]
Hey I added the option in chat formatting so that it can be suppressed now.

File: js/client-topbar.js [hunk 1]
```diff
@@ -593,7 +593,8 @@
 			buf += '<p><label class="optlabel"><input type="checkbox" name="italics" ' + (cur.hideitalics ? 'checked' : '') + ' /> Suppress __<em>italics</em>__</label></p>';
 			buf += '<p><label class="optlabel"><input type="checkbox" name="monospace" ' + (cur.hidemonospace ? 'checked' : '') + ' /> Suppress ``<code>code formatting</code>``</label></p>';
 			buf += '<p><label class="optlabel"><input type="checkbox" name="strikethrough" ' + (cur.hidestrikethrough ? 'checked' : '') + ' /> Suppress ~~<s>strikethrough</s>~~</label></p>';
-			buf += '<p><label class="optlabel"><input type="checkbox" name="superscript" ' + (cur.hidesuperscript ? 'checked' : '') + ' /> Suppress ^^<sup>superscript</sup>^^</label></p>';
+		    buf += '<p><label class="optlabel"><input type="checkbox" name="superscript" ' + (cur.hidesuperscript ? 'checked' : '') + ' /> Suppress ^^<sup>superscript</sup>^^</label></p>';
```
Comment (Zarel) [inline js/client-topbar.js]: Indentation.

**Comment by CheeseMuffin** [comment]
Sorry, fixed

**Comment by Zarel** [comment]
Looks good! Thanks a lot!

**Pull request merged**